ReactWindows/ReactNative.Shared/Common/IntegerHelpers.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace ReactNative.Common
+{
+    static class IntegerHelpers
+    {
+        public static uint LittleEndianToHost(uint value)
```

Comment:
could we call this InetHelpers (or something similar) to reference the C header whose API we're emulating, and call the static method the same thing?